feature-detects/event/oninput.js

```diff
@@ -7,10 +7,10 @@
     "href": "https://developer.mozilla.org/en-US/docs/Web/API/GlobalEventHandlers.oninput"
   },{
     "name": "WHATWG spec",
-    "href": "https://html.spec.whatwg.org/multipage/forms.html#common-input-element-attributes"
+    "href": "https://html.spec.whatwg.org/multipage/input.html#common-input-element-attributes"
   },{
-    "name": "Detecting onInput support",
-    "href": "http://danielfriesen.name/blog/2010/02/16/html5-browser-maze-oninput-support"
+    "name": "Undetectable",
+    "href": "https://github.com/Modernizr/Modernizr/wiki/Undetectables"
```

Comment:
maybe point to the issue mentioned in the wiki directly? https://github.com/Modernizr/Modernizr/issues/210

Reply:
Good point!